### ISSUE TITLE
alert system

### ISSUE THREAD

**Issue by MesterMan03** [Mester-Ulquiorra/web#6]
**Goal**: provide a one-directional (server -> client) way of sending alerts and other types of data, either real-time or async.
 **Types**
`AlertType` = "appeal" | "punishment"
`AlertMessage<T extends AlertType>` =
1.	type: same as T
2.	data: AlertData<T>

`AlertData<"appeal">` (used for appeals that have been processed by the staff team) =
1. status: "accepted" or "rejected"
2. reason: string

`AlertData<"punishment">` (used for new punishments, always in real-time) = same as the punishment object from `/user/punishments`

**Example**
```
{
  "type": "appeal",
  "data": {
    "status": "accepted",
    "reason": "thx for the bribe"
  }
}
```

**Route changes**
`/user` may return a new `alerts` field with type `AlertMessage<unknown>[]` when there are unread alerts.
`/sse` returns a Server-Sent Events stream which may occasionally send a `alert` event with a `AlertMessage<unknown>` object.

**Comment by MesterMan03** [comment]
**API update**
`/sse` now returns an `appeal` alert in production mode.

**Comment by MesterMan03** [comment]
Closed in favour of #8
